torch/accelerator/__init__.py

```diff
@@ -30,6 +30,8 @@ def device_count() -> int:
     Returns:
         int: the number of the current :ref:`accelerator<accelerators>` available.
             If there is no available accelerators, return 0.
+
+    .. note:: This API may cause issues in forked processes. See :ref:`multiprocessing-poison-fork-note` for more details.
```

Comment:
I would have two feedback:
- Rather than adding this on almost every method, we should just mention here which method doesn't actually poison the fork
- Either way, we should be clearer in the message: This API causes the runtime to be initialized and thus fork cannot be used after this API has been called.

Reply:
Updated.